tests/cli/test_version_check.py

```diff
@@ -242,8 +263,11 @@ def pypi_server(fake_pypi: Tuple[FakePyPI, Dict[str, int]]) -> FakePyPI:
     async with VersionChecker(
         _PyPIVersion.create_uninitialized(), connector=connector
     ) as checker:
-        version = await checker._fetch_pypi("neuromation")
+        payload = await checker._fetch_pypi("neuromation")
+        version = checker._parse_max_version(payload)
         assert version == pkg_resources.parse_version("0.0.0")
+        upload_time = checker._parse_version_upload_time(payload, version)
+        assert upload_time == date.min
```

Comment:
In this PR, we miss a test that checks that when `await checker.run()` is called, then indeed all versions get updated. Will be in a consequent PR (already implemented).